### PR TITLE
Update first-app-lesson-11.md

### PR DESCRIPTION
**Pull request by Hadi2525**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

The existing document results in an error. The error indicates that the anchor element doesn't have the binding property for routerLink.

The corresponding module (RouterModule) should be imported in the component template.

Issue Number: N/A

## What is the new behavior?
The updated document now addresses the content in the tutorial and runs without any errors.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/tutorial/first-app/first-app-lesson-11.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ In this case, `:id` is dynamic and will change based on how the route is request
     <code-example header="Add anchor with a routerLink directive to housing-location.component.ts" path="first-app-lesson-11/src/app/housing-location/housing-location.component.ts" region="add-router-link"></code-example>
 
     The `routerLink` directive enables Angular's router to create dynamic links in the application. The value assigned to the `routerLink` is an array with two entries: the static portion of the path and the dynamic data.
+    
+    For the `routerLink` to work in the template, add `RouterModule` at the file level in `housing-location` component and update the `imports` in `@Component` as well.
```
Comment (jessicajaniuk) [inline aio/content/tutorial/first-app/first-app-lesson-11.md:47]: I'm not sure if this lesson is set up as standalone or not, but if it's standalone, you shouldn't need to import the whole RouterModule here. @MarkTechson, do you want to chime in here?

Comment (Hadi2525) [reply aio/content/tutorial/first-app/first-app-lesson-11.md:47]: I am following the steps in the tutorial, this component is indeed standalone, however, not importing the related module does not pass the property to the anchor element.

Comment (jessicajaniuk) [reply aio/content/tutorial/first-app/first-app-lesson-11.md:47]: Yes, but you don't need to import the entire RouterModule for the RouterLink to work. I think the right fix here is to import the relevant needs from the Router, and not the entire RouterModule.

Comment (MarkTechson) [reply aio/content/tutorial/first-app/first-app-lesson-11.md:47]: This is a great callout, @Hadi2525. The source code features the correct imports but there isn't a callout for them in the tutorial.

Can you update this to be: 

For the `routerLink` to work in the template, add a file level import of `RouterLink` and `RouterOutlet` from `'@angular/router'`, then update the component `imports` array to include both `RouterLink` and `RouterOutlet`

Comment (Hadi2525) [reply aio/content/tutorial/first-app/first-app-lesson-11.md]: Thank you @MarkTechson for your update! I am glad it helps :)

I see other inconsistencies within the tutorial as well. I will send corrections for them as well.

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — COMMENTED** [review]
LGTM

**Review by MarkTechson — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
@Hadi2525 Looks like CI is failing for lint. Can you update your commit message to fix this and then force push? Note: the commit message is _different_ from the PR header.

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit 9a172a934502aa6f8c656825e81cd5a08bde30b7.

**Pull request closed without merging**